lib/internal/repl/await.js

```diff
@@ -92,13 +100,40 @@ for (const nodeType of ObjectKeys(walk.base)) {
 }
 
 function processTopLevelAwait(src) {
-  const wrapped = `(async () => { ${src} })()`;
+  const wrapPrefix = '(async () => { ';
+  const wrapped = `${wrapPrefix}${src} })()`;
   const wrappedArray = ArrayFrom(wrapped);
   let root;
   try {
     root = parser.parse(wrapped, { ecmaVersion: 'latest' });
-  } catch {
-    return null;
+  } catch (e) {
+    if (e.message.startsWith('Unterminated '))
```

Comment:
```suggestion
    if (StringPrototypeStartsWith(e.message, 'Unterminated '))
```

Reply:
Thanks I will remember these one day :)

Reply:
maybe one day JS will have an ergonomic way to do this (－‸ლ)